sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.7.0b3 (Unreleased)
+## 1.7.0b3 (2020-08-10)
 
 ### Features Added
```

Comment:
Almost forgot -- I think this empty heading might raise an error in the release pipeline